openshift_metrics/merge.py

```diff
@@ -67,21 +76,96 @@ def get_su_definitions(report_month) -> dict:
     return su_definitions
 
 
+def load_and_merge_metrics(interval_minutes, files: List[str]) -> MetricsProcessor:
+    """Load and merge metrics
+
+    Loads metrics from provided json files and then returns a processor
+    that has all the merged data.
+    """
+    processor = MetricsProcessor(interval_minutes)
+    for file in files:
+        with open(file, "r") as jsonfile:
+            metrics_from_file = json.load(jsonfile)
+            cpu_request_metrics = metrics_from_file["cpu_metrics"]
+            memory_request_metrics = metrics_from_file["memory_metrics"]
+            gpu_request_metrics = metrics_from_file.get("gpu_metrics", None)
+            processor.merge_metrics("cpu_request", cpu_request_metrics)
+            processor.merge_metrics("memory_request", memory_request_metrics)
+            if gpu_request_metrics is not None:
+                processor.merge_metrics("gpu_request", gpu_request_metrics)
+    return processor
+
+
+def load_metadata(files: List[str]) -> MetricsMetadata:
```

Comment:
I think I could load data and metadata in a single loop instead of loading files twice. And for that I reason I don't like what I've done. I am going to refactor it again later.